### PR TITLE
Feature/improve build window scene selector

### PR DESCRIPTION
**Pull request by MichaelLiminal**
Users now use a GUI object field to select which scene should be built and not a file browser as it's more user-friendly.

### CODE REVIEW

File: SDK/Editor/Builder/Windows/BuildWindow.cs [hunk 1]
```diff
@@ -55,25 +55,29 @@ public override void Draw(BuildWindowConfig config)
...
-                    scenePath = EditorUtility.OpenFilePanelWithFilters("Scene Finder", scenePath, new string[] { "FileType", "unity" });
-                    scenePath = DirectoryUtils.ReplaceBackWithForwardSlashes(scenePath);
+                    scenePath = AssetDatabase.GetAssetPath(_targetScene);
                 }
             }
             EditorGUILayout.EndHorizontal();
         }
 
         private BuildPlatform _selectedPlatform;
+        private SceneAsset _targetScene;
```
Comment (cschladetsch) [inline SDK/Editor/Builder/Windows/BuildWindow.cs]: All fields should precede all methods.

Comment (MichaelLiminal) [reply SDK/Editor/Builder/Windows/BuildWindow.cs]: I thought we were supposed to maintain existing styles when modifying existing scripts?

Tin had put 'private BuildPlatform _selectedPlatform;' below the methods when the script was written.

Comment (cschladetsch) [reply SDK/Editor/Builder/Windows/BuildWindow.cs]: I'll change the Policy.

What was meant was: Follow convention of the local file, if it's an external file. If it's an internal file, follow convention of the local file and make a separate PR for the formatting change.

**Comment by cschladetsch** [comment]
Policy Change:

Do follow syntax and formatting conventions used in any file you modify, even if doing so contravenes conventions stated in this document.
Follow convention of the local file, if it's an external file. 
If it's an internal file, follow convention of the local file and make a separate PR for the formatting change.

**Pull request merged**